interfaces/builtin/steam_support.go

```diff
@@ -150,6 +150,7 @@ mount options=(rw, rbind) /oldroot/etc/group -> /newroot/etc/group,
 mount options=(rw, rbind) /oldroot/etc/passwd -> /newroot/etc/passwd,
 mount options=(rw, rbind) /oldroot/etc/host.conf -> /newroot/etc/host.conf,
 mount options=(rw, rbind) /oldroot/etc/hosts -> /newroot/etc/hosts,
+mount options=(rw, rbind) /oldroot/usr/share/zoneinfo/** -> /newroot/etc/localtime,
```

Comment:
Mmm... is this correct? I mean: /etc/localtime is a soft link to a file or folder inside /usr/share/zoneinfo... shouldn't there be two entries? One pointing /etc/localtime from the old root into the new root, and another for the /usr/share/zoneinfo?

Reply:
No, it doesn't need that.  The issue this resolves is the proton runtime resolves the link and if the link is under /usr it does the right thing for us.  On core desktop that link is 2 deep, so that doesn't resolve in the way we want.  For reference: https://gitlab.steamos.cloud/steamrt/steam-runtime-tools/-/blob/main/pressure-vessel/runtime.c#L3763-3808

Reply:
Then, LGTM.